test-harness/src/test/java/io/jenkins/plugins/casc/impl/configurators/MissingConfiguratorTest.java

```diff
@@ -21,6 +22,8 @@ public class MissingConfiguratorTest {
             message = "No hudson.security.AuthorizationStrategy implementation found for globalMatrix")
     @Test
     public void testThrowsSuggestion() {
+        // The conditions for this test can be false in PCT runs
+        assumeThat(j.jenkins.getPlugin("matrix-auth"), nullValue());
```

Comment:
PCT runs with all plugins in the megawar, including `matrix-auth`, so skip this test in that context. It still runs in the context of this repository's CI build.